app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx

```diff
@@ -1,17 +1,40 @@
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import { injectIntl, defineMessages } from 'react-intl';
 import { LaunchExecutionChart } from './launchExecutionChart';
 import { IssueStatisticsChart } from './issueStatisticsChart';
 import styles from './launchExecutionAndIssueStatistics.scss';
 
 const cx = classNames.bind(styles);
+const launchNameBlockHeight = 40;
 
-export const LaunchExecutionAndIssueStatistics = (props) => (
-  <div className={cx('launch-execution-and-issues-chart')}>
-    <LaunchExecutionChart {...props} />
-    <IssueStatisticsChart {...props} />
-  </div>
-);
+const localMessages = defineMessages({
+  launchNameText: {
+    id: 'LaunchExecutionAndIssueStatistics.launchNameText',
+    defaultMessage: 'Launch name:',
+  },
+});
+
+export const LaunchExecutionAndIssueStatistics = injectIntl((props) => {
+  const { result = {} } = props.widget.content;
+  const { name, number } = result[0];
+  const launchName = number ? `${name} #${number}` : name;
```

Comment:
Doesn't work for case when `number` is 0

Reply:
Could case number be 0?

Reply:
Clarified with BA - launch number cannot be 0